content/efcore/getting-started/samples/querying/filtering-entities/Program.cs

```diff
@@ -11,10 +11,11 @@ public static void Main()
             var books = context.Books
                 .Where(b => b.Genre == "Historical")
                 .ToList();
-
+			
+			Console.WriteLine("Historical books in library -");
```

Comment:
Colons are used to introduce lists, not dashes:

- http://www.wheaton.edu/Academics/Services/Writing-Center/Writing-Resources/Semicolons-Colons-and-Dashes
- http://writingcenter.unc.edu/handouts/semi-colons-colons-and-dashes/
- https://www.pugetsound.edu/academics/academic-resources/cwlt/writing-advisor-schedule/writing-resources/colon-and-dashes/
- https://www.bethel.edu/undergrad/academics/support/colon-dash.pdf